### PR TITLE
Restart from stage

### PR DESCRIPTION
**Pull request by skaspi**
Hi,

This is a part of the following ticket: https://projects.engineering.redhat.com/browse/RHEVM-5503

Please CR + merge

david

### CODE REVIEW

File: vars/other.groovy [hunk 1]
```diff
@@ -1,4 +1,30 @@
-def map =
+def get_current_stage(start_stage, only_stage){
```
Comment (KKoukiou) [inline vars/other.groovy]: Missing space after closing parenthesis.

Comment (KKoukiou) [inline vars/other.groovy]: It's good practice to add docstrings, (or how they are called in groovy) to explain what the arguments are.
What is only_stage actually? To re-run only that stage specified by start_stage? It doesn't do that currently.

File: vars/other.groovy [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+               'Configure ESX hosts':true,
+               'Create transformation mappings':true,
+               'Create transformation plans':true,
+               'Start performance monitoring':true,
+               'Execute transformation plans':true,
+               'Monitor transformation plans':true,
+               'Stop performance monitoring':true]
+
+  for (e in stages) {
+    if (e.key == start_stage ) {
```
Comment (KKoukiou) [inline vars/other.groovy]: The following two if lines can be combined into one negating the condition and executing break.
Like:
```
if (e.key == start_stage && !only_stage)
    break;
```

**Comment by KKoukiou** [comment]
@skaspi in addition you have 3 commits, the first two are redundant. Please remove them before next push.

File: vars/other.groovy (hunk 1 above)
Comment (skaspi) [reply vars/other.groovy]: I can add a docstrings to this function. But what about all other functions ? You want me to document them also?

Comment (skaspi) [reply vars/other.groovy]: Done

File: vars/other.groovy (hunk 2 above)
Comment (skaspi) [reply vars/other.groovy]: Your solution is wrong in case when only_stage=True .

Comment (skaspi) [reply vars/other.groovy]: Your solution is wrong in case when only_stage=True.
I will leave it as is.

File: vars/other.groovy (hunk 1 above)
Comment (KKoukiou) [reply vars/other.groovy]: Of course no, only what this PR introduces.

**Pull request closed without merging**